osu.Game/Overlays/Direct/PanelDownloadButton.cs

```diff
@@ -14,12 +15,12 @@ public class PanelDownloadButton : BeatmapDownloadTrackingComposite
     {
         protected bool DownloadEnabled => button.Enabled.Value;
 
-        private readonly bool noVideo;
+        private readonly bool? noVideo;
 
         private readonly ShakeContainer shakeContainer;
         private readonly DownloadButton button;
 
-        public PanelDownloadButton(BeatmapSetInfo beatmapSet, bool noVideo = false)
+        public PanelDownloadButton(BeatmapSetInfo beatmapSet, bool? noVideo = null)
```

Comment:
I'm a little perplexed by this. As far as I can see, there isn't a single usage where `noVideo` is ever explicitly provided, it's all defaults or delegating ctors. Therefore I don't really know what the `null` value is supposed to convey - could you explain this choice? xmldoc would also be helpful for future travellers if this stays.

Reply:
I would remove the param and have the setting be the only thing that matters. Wasn't sure if this was fine.